examples/file/download_file.py

```diff
@@ -0,0 +1,30 @@
+"""Example to download a file from SystemLink."""
+
+from shutil import copyfileobj
+
+from nisystemlink.clients.file import FileClient
+
+client = FileClient()
+
+file_id = "<Id of file to download>"
+
+# Fetch the file metadata to get the name
+files = client.get_files(file_ids=file_id)
+
+if not files.available_files:
+    raise Exception(f"File ID {file_id} not found.")
+
+
+file_name = "Untitled"
+
+file_properties = files.available_files[0].properties
+
+if file_properties:
+    file_name = file_properties.get("Name", "Untitled")
```

Comment:
I don't personally like the `.get` method on dictionaries. Can we use `file_properties["Name"]` instead?